lua/lspconfig/ltex.lua

```diff
@@ -0,0 +1,33 @@
+configs.ltex = {
+  default_config = {
+    cmd = { "ltex-ls" },
+    filetypes = { "tex", "bib", "markdown" },
+    settings = {
```

Comment:
`root_dir` seems to be missing. It used to be like this, but not sure if it's a good one for this LSP (I don't use it)

```suggestion
     root_dir = function(filename)
       return util.path.dirname(filename)
     end,
    settings = {
```

Reply:
it should just be a git fallback until I add scratch server support

Reply:
> but not sure if it's a good one for this LSP (I don't use it)

@williamboman this applies to all lsp4j based ones (ltex-ls, jdtls, etc):

- https://github.com/eclipse/lsp4j/blob/40a1fe44015626f8ea63dd7ed36ebbd5a1ce84f8/org.eclipse.lsp4j/src/main/java/org/eclipse/lsp4j/services/LanguageClient.java#L116-L118